sound/hda/intel-dsp-config.c

```diff
@@ -424,8 +424,10 @@ static const struct config_entry *snd_intel_dsp_find_config
 			int i;
 
 			for (i = 0; i < table->codec_hid->num_codecs; i++)
-				if (!acpi_dev_present(table->codec_hid->codecs[i], NULL, -1))
-					continue;
+				if (acpi_dev_present(table->codec_hid->codecs[i], NULL, -1))
+					break;
+			if (i == table->codec_hid->num_codecs)
+				continue;
```

Comment:
@plbossart Should this be squashed into the previous commit?

Reply:
yes indeed, nice catch, I squashed a fixup in the wrong file. Thanks for spotting this @bardliao, fixed now.